jgrapht-ext/src/main/java/org/jgrapht/ext/DOTUtils.java

```diff
@@ -22,25 +22,30 @@
 import org.jgrapht.*;
 
 /**
+ * Class with DOT format related utilities.
+ * 
  * @author Christoph Zauner
+ * 
+ * @deprecated in favor of implementing the current functionality directly by the user
  */
+@Deprecated
 public class DOTUtils
```

Comment:
Hi, @d-michail. I think that this class shouldn't be deprecated although now it have only the utility method to convert the graph (I agree that `convertGraphToDotString()` should be deprecated).
But you can see my PR (#286) to have an idea of what I mean regarding why this shouldn't be deprecated. This class (or a new one named `DOTLanguageUtils` could contain interesting methods for the language validation (for instance, ID validation using pre-compiled patterns) and/or keyword definitions (__strict__, __graph__, __digraph__).

What I tried in my PR is use the `DOTUtils` class as a container to code this language definitions to be use in the importer/exporter instead of hard-coded them. If the language change, this allow easy maintenance of the code and keeping in sync language changes. In addition, API users could benefit for this variables if they want to implement their own classes to handle the language.